src/Plugin/Filter/UrlEmbedFilter.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\url_embed\Plugin\Filter\UrlEmbedFilter.
+ */
+
+namespace Drupal\url_embed\Plugin\Filter;
+
+use Drupal\Core\Annotation\Translation;
+use Drupal\Component\Utility\Html;
+use Drupal\filter\Annotation\Filter;
+#use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\filter\Plugin\FilterBase;
+#use Symfony\Component\DependencyInjection\ContainerInterface;
+
+include '../../Embed/src/autoloader.php';
```

Comment:
This should be left out. We shouldn't need to load things if we're using the Composer Manager module.